src/croniter/croniter.py

```diff
@@ -186,6 +186,9 @@ class CroniterNotAlphaError(CroniterBadCronError):
 
 class croniter(object):
     MONTHS_IN_YEAR = 12
+
+    # This helps with expanding `*` fields into `lower-upper` ranges. Each item
+    # in this tuple maps to the field index of YEAR_FIELDS.
```

Comment:
comment partially wrong,  `s/ field index of YEAR_FIELDS/corresponding field index/` would be more appropriate as it is used by any form of cron no matter unix, second, or year one.

Reply:
Yeah makes sense. My intention was to say that it includes everything up to the YEAR_FIELDS, but this is probably a bit less confusing.

Updated